### PR TITLE
Add marine verification figure display capability (HTML)

### PR DESCRIPTION
**Pull request by KatherineLukens-NOAA**
This PR adds the capability to display marine verification figures via an internal HTML document. The `README` file lists the steps one should follow to utilize the `*.html` file.

The files that should be reviewed are:
```
index_vrfy_marine.html
README
```

### CODE REVIEW

**Review by guillaumevernieres — CHANGES_REQUESTED** [review]
@KatherineLukens-NOAA , we can't commit binary files, remove all the figures from your branch.

File: utils/soca/fig_gallery/marine_vrfy_display/README [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+To use, follow these steps:
+
+----------------------------
+
+1. Login to Hera via X2Go and type on the command line:
+
+        firefox index_vrfy_marine.html
+
+2. In the webpage,
+	a. Enter the path to your /vrfy directory.
```
Comment (guillaumevernieres) [inline utils/soca/fig_gallery/marine_vrfy_display/README]: that's not correct anymore. Path to something that comes out of hpss?

**Review by apchoiCMD — DISMISSED** [review]
Excited-

File: utils/soca/fig_gallery/marine_vrfy_display/index_vrfy_marine.html [hunk 2]
```diff
@@ -0,0 +1,853 @@
...
+					</div>
+			        </div>
+			    </div>
+                            
+                        </ul>
+		</div>
+	</div>
+	<div id="main">
+		<h1><font color="blue">Marine Figures</font></h1>
+		<h2>Add path to your /vrfy directory:</h2>
```
Comment (guillaumevernieres) [inline utils/soca/fig_gallery/marine_vrfy_display/index_vrfy_marine.html]: update this, maybe give an example?

Comment (KatherineLukens-NOAA) [reply utils/soca/fig_gallery/marine_vrfy_display/index_vrfy_marine.html]: Sure I will do that

File: utils/soca/fig_gallery/marine_vrfy_display/README (hunk 1 above)
Comment (KatherineLukens-NOAA) [reply utils/soca/fig_gallery/marine_vrfy_display/README]: I added an input box where the user declares if the vrfy figures are in their COMROOT or from HPSS. This flag determines the internal path that HTML uses to find and display the figures.

File: utils/soca/fig_gallery/marine_vrfy_display/index_vrfy_marine.html (hunk 2 above)
Comment (KatherineLukens-NOAA) [reply utils/soca/fig_gallery/marine_vrfy_display/index_vrfy_marine.html]: I added some clarifying text about what should be included in the path, depending on whether the vrfy figures are in the user's COMROOT or from HPSS.

**Review by JohnSteffen-NOAA — DISMISSED** [review]

**Review by guillaumevernieres — APPROVED** [review]
:+1:

**Pull request merged**